subsys/app_memory/Kconfig

```diff
@@ -0,0 +1,9 @@
+
+menu "General Kernel Options"
+config APP_SHARED_MEM
+	bool
+	prompt "alt application shared memory"
+	default n
```

Comment:
We obviously will want a better description for this option ultimately, maybe "Support fine-grained application memory sharing" or something.